dist/.ddev/wunderio/core/tooling-grumphp.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+#
+# Helper script to run GrumPHP.
+#
+
+set -exu
+export PATH=/bin:/sbin:/usr/bin:/usr/sbin:/usr/local/bin:/var/www/html/vendor/bin
+
+php /var/www/html/vendor/bin/grumphp "$@"
```

Comment:
What's the idea behind having a script instead of running `php /var/www/html/vendor/bin/grumphp "$@"` directly in the `commands/web/....`?

Reply:
Maybe this is not the best example, but it's because we could override the command if needed. Better examples would be build script or post db hooks. You might want to do totally different thing in your project what the default is.

In the commands I'm not running this script directly, but through _run-scripts.sh
`/var/www/html/.ddev/wunderio/core/_run-scripts.sh tooling-grumphp.sh "$@"`

_run-scripts.sh runs all scripts from
`.ddev/wunderio/core/`
folder, but if the same script exists under 
`.ddev/wunderio/custom/`,
then it'll run it from there.

If you update this project, everything under core is updated if something is coming from upstream. Custom folder is not. 

Grumphp example would be that somebody wants more verbosity from the Grumphp, so they would copy the original script, add the parameter, and it would be always there, specific to that project. Without this you'd need to create grumphp-custom or some other tooling.

Reply:
I see - so we would copy `tooling-[...].sh` script we want to override to `custom` folder and do our modifications?

Reply:
Yes, exactly. @artursv